src/pymodaq/extensions/bayesian/bayesian_optimisation.py

```diff
@@ -662,7 +666,7 @@ def main(init_qt=True):
 
     dashboard = DashBoard(area)
     daq_scan = None
-    file = Path(get_set_preset_path()).joinpath(f"{'beam_steering_mock'}.xml")
```

Comment:
for this main function, use the mkapp method from pymodaq_gui to simplify the initialization and avoid bugs! check on the daq_scan extension if needed!